api/helpers/economy/give-to-player.js

```diff
@@ -62,8 +62,8 @@ module.exports = {
 
             return exits.success();
         } catch (error) {
-            sails.log.error(`HELPER economy:give-to-player - ${error}`);
-            return exits.error(error);
+            //sails.log.error(`HELPER economy:give-to-player - ${error}`);
```

Comment:
Why disable the error handling here? This will cause the helper to hang indefinetly if something bad happens

Reply:
Wasn't intended. I think I disabled it when I was debugging and testing to get familiar with some other things and I forgot to put it back.